packages/smithy-client/src/date-utils.spec.ts

```diff
@@ -36,6 +41,80 @@ describe("parseRfc3339DateTime", () => {
     "1985-04-05T23:61:50.52Z",
     "1985-04-05T23:20:61.52Z",
     "1985-04-31T23:20:50.52Z",
+    "1996-12-19T16:39:57",
+    "2019-12-16T22:48:18+02:04",
+    "2019-12-16T22:48:18-01:02",
+    "2005-02-29T15:59:59Z",
```

Comment:
nit: move this to after line 43 (`1985-04-31T23:20:50.52Z`) like it was earlier, to keep the grouping of cases of values being beyond the range of month/min/sec together. it reduces the overhead on a future reader to understand what the different cases are covering.

Reply:
Fixed this ordering.